### ISSUE TITLE
the performance about PS-ViT-Ti/14

### ISSUE THREAD

**Issue by ytoon** [yuexy/PS-ViT#4]
Hi,

I have trained PS-ViT-Ti/14 following the default setting in the code.  But the performance is only 74.6, which is lower than 75.6 in the paper. Could you show more training details?

**Comment by yuexy** [comment]
Hi, can you provide more details? such as the batch size you used and the number of GPUs.

**Comment by ytoon** [comment]
I used this [config](https://github.com/yuexy/PS-ViT/blob/master/configs/ps_vit_ti_14.yaml). The number of GPUs is 8. 
In addition, I also tried the following setting:
```
model: "ps_vit_ti_14" 
batch_size: 64
lr: 5e-4
weight_decay: 0.05
img_size: 224
warmup_epochs: 5
```
But these settings have the same performace ~74.6.

**Comment by yuexy** [comment]
The total batch size should be 512, and the weight decay should be 0.03.
I have updated configuration files, pls fetch the latest version and try again.

**Comment by yuexy** [comment]
Thanks for your feedback. I guess I can close this issue now.

**Comment by ytoon** [comment]
Thank you. I got it. The PS-ViT-Ti/14's performance is 75.6, which is the same as the paper.
